src/main/java/com/zachary_moore/lint/Error.java

```diff
@@ -0,0 +1,46 @@
+package com.zachary_moore.lint;
+
+import java.util.Objects;
+
+
+public class Error {
+
+  private final String path;
+
+  private final String message;
+
+  public Error(String path, String message) {
```

Comment:
Lets swap this to builder pattern since we might add multiple things to this, eg, line number / some might be optional if user wants to report or not